### PR TITLE
Moved GetAllRolesHandler from control-examples to control-admin.

### PR DESCRIPTION
**Pull request by tsallinen**

### CODE REVIEW

File: control-admin/src/main/java/fi/nls/oskari/control/admin/GetAllRolesHandler.java [hunk 1]
```diff
@@ -33,7 +38,14 @@ public void init() {
     }
 
     @Override
-    public void handleAction(ActionParameters params) throws ActionException {
+    public void preProcess(ActionParameters params) throws ActionException {
+        if (!params.getUser().isAdmin()) {
```
Comment (jampukka) [inline control-admin/src/main/java/fi/nls/oskari/control/admin/GetAllRolesHandler.java]: Could have used ActionParameters#requireAdminUser();

Also could move the userService == null check from handleGet() to preProcess()

Comment (tsallinen) [reply control-admin/src/main/java/fi/nls/oskari/control/admin/GetAllRolesHandler.java]: Updated PR as suggested.

**Comment by jampukka** [comment]
Looks good

**Comment by ZakarFin** [comment]
Thanks, control-admin is a more logical place for the functionality.

**Pull request merged**